romancal/regtest/test_tweakreg.py

```diff
@@ -81,6 +81,50 @@ def test_tweakreg(
 
     assert rms < 1.3 / np.sqrt(2)
 
+    # check if the Mean Absolute Error is less that 10 milliarcsec  (DMS406)
+    abs_diff = (np.absolute(diff)) / 1.0e-3
+    mean_abs_error = (tweakreg_out.meta.wcs_fit_results.mae) / 1.0e-3
+    assert mean_abs_error < 10.0
+    passmsg = "PASS" if mean_abs_error < 10.0 else "FAIL"
+    assert np.max(abs_diff) < 10.0
+    passmsg = "PASS" if np.max(abs_diff) < 10.0 else "FAIL"
+    dms_logger.info(
+        f"DMS406 {passmsg} the Absolute Astrometric Uncertainty of {np.max(abs_diff):5.2f} mas is less that 10 mas."
+    )
+
+    # check that tweakreg has been run
+    if tweakreg_out.meta.cal_step.tweakreg == "COMPLETE":
+        # Find the reference catalog used by tweakreg
+        for entry in tweakreg_out.meta.cal_logs:
+            refcat_name = " "
+            if "abs_refcat" in entry:
+                log_substring = entry[entry.rfind("abs_refcat") :]
+                refcat_name = log_substring[: log_substring.find("\n")]
+
+        assert tweakreg_out.meta.cal_step.tweakreg == "COMPLETE"
```

Comment:
```suggestion
    if tweakreg_out.meta.cal_step.tweakreg == "COMPLETE":
        # Find the reference catalog used by tweakreg
        for entry in tweakreg_out.meta.cal_logs:
            refcat_name = " "
            if "abs_refcat" in entry:
                log_substring = entry[entry.rfind("abs_refcat") :]
                refcat_name = log_substring[: log_substring.find("\n")]

        assert tweakreg_out.meta.cal_step.tweakreg == "COMPLETE"
```
Would you help me understand what this test is expecting? Why is there a test `if tweakreg... == COMPLETE` that then contains an assert that `tweakreg... == COMPLETE`?

Reply:
removed assert